src/assets/scss/style.scss

```diff
@@ -0,0 +1,141 @@
+/* Change styles for span and cancel button on extra small screens */
+$button-color: #4caf50;
+$border-color: #f1f1f1;
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  background-color: #d3d3d3;
+}
+form {
+  border: 3px solid $border-color;
+}
+input[type="email"] {
```

Comment:
Since CSS for both email and password is same then no need to duplicate it, and also follow SCSS format for hierarchy like 
form {
 border: 3px solid;

 input {
  //css for input
 } 
}